packages/manager/modules/manager-pci-common/src/components/rclone-download/StorageRegionSelector.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useTranslatedMicroRegions } from '@ovh-ux/manager-react-components';
+import { OdsSelect } from '@ovhcloud/ods-components/react';
+import { TRegion } from '../../api/data';
+
+interface StorageRegionSelectorProps {
+  regions: TRegion[];
+  onRegionChange: (regionName: string) => void;
+}
+
+export function StorageRegionSelector({
+  regions,
+  onRegionChange,
+}: Readonly<StorageRegionSelectorProps>) {
+  const { translateMicroRegion } = useTranslatedMicroRegions();
+
+  return (
+    <OdsSelect
+      defaultValue={regions?.[0]?.name}
+      name="currentRegion"
+      data-testid="regions_select"
+      onOdsChange={(event) => {
+        if (!event?.detail?.value) return;
+        onRegionChange(`${event.detail.value}`);
+      }}
+    >
+      {regions?.map((region: TRegion, index: number) => (
+        <option key={`${region.name}-${index}`} value={region.name}>
```

Comment:
`region.name` isn't it already unique? So concatenation with the index is not required.

Reply:
done thanks